pkgs/applications/networking/instant-messengers/rambox/env.js

```diff
@@ -0,0 +1 @@
+var auth0Cfg={clientID:"y9am0DVawe2tvlA3ucD7OufpJHZZMjsO",domain:"rambox.auth0.com"};
```

Comment:
are we allowed to put this client ID in our repo?

Reply:
No, not explicitly.

A new client ID requires an account at https://auth0.com/. I could register one on my own but I'm not sure if that's appropriate. Chromium derivation also has API credentials:

https://github.com/NixOS/nixpkgs/blob/e8f1ddcbd1d9bd0f523670cc9e7c0f69ba75c243/pkgs/applications/networking/browsers/chromium/common.nix#L197-L203

Are they registered using a special email address or is it just someone who happens to maintain the derivation?

Reply:
Nevermind, used my own account for now. Fixed!